AODVv2-pandoc-edits/AODVv2.mkd

```diff
@@ -471,17 +471,17 @@ See section [](#suppressrerr) for instructions on how to update the set.
 Metrics measure a cost or quality associated with a route or a link, e.g., latency, delay, financial cost, energy, etc. Metric
 values are reported in Route Request and Route Reply messages.
 
-In Route Request messages, the metric describes the cost of the route from OrigPrefix to the router sending the Route Request. For RREQ_Gen, this is the cost associated with the Router Client Set entry which includes OrigAddr. For routers which forward the RREQ, this is the cost from OrigPrefix to the forwarding router, combining the metric value from the received RREQ message with knowledge of the link cost from the sender to the receiver, i.e., the incoming link cost. This updated route cost is included when forwarding the Route Request message,
+In Route Request messages, the metric describes the cost of the route from OrigPrefix to the router sending the Route Request. For RREQ_Gen, this is the metric associated with the Router Client Set entry which includes OrigAddr. For routers which forward the RREQ, this is the metric from OrigPrefix to the forwarding router, combining the metric value from the received RREQ message with knowledge of the link metric from the sender to the receiver, i.e., the incoming link metric. This updated route metric is included when forwarding the Route Request message,
```

Comment:
I think "For RREQ_Gen, this is the..." should still be "For RREQ_Gen, this is the *cost* associated with...", since that sentence is part of the explanation what the metric actually is